js/background.js

```diff
@@ -1,10 +1,24 @@
 
 const TIMER_NAME = "timer";
 
-var runRefresh = function(cfg) {
+function loadTabDetailsAfterLoad(tab, callback) {
+  chrome.tabs.query({windowId: tab.windowId, index: tab.index}, function (tabs) {
+    const ntab = tabs.pop();
```

Comment:
Assuming tabs is an array, I dislike mixing mutable style and functional style. afaik, pop is mutable on tabs array, hence I personally would prefer tabs.pop();
A better option may be array destructuring.